test/Microsoft.Azure.Devices.Edge.Test/helpers/SasManualProvisioningFixture.cs

```diff
@@ -13,6 +11,16 @@ public class SasManualProvisioningFixture : ManualProvisioningFixture
     {
         protected EdgeRuntime runtime;
 
+        public SasManualProvisioningFixture()
```

Comment:
I think this empty constructor is unnecessary

Reply:
When I remove this, compiler complains for other tests that were using SasManualProvisioningFixture without specifying the parameters.

Because I added a constructor with parameters in the ManualProvisioningFixture, C# won't create a default, paramater-less constructor automatically anymore. So I either need to keep this or add constructors with parameters in all the classes that use it. (for example adding a constructor to Module.cs with connection string and event hub endpoint string)

(that is my best guess at what's happening)

Reply:
Well just ignore me today. I don't know what I'm talking about...